MyConfig.h

```diff
@@ -639,6 +639,10 @@
 #define MY_MAC_ADDRESS 0xDE, 0xAD, 0xBE, 0xEF, 0xFE, 0xED
 #endif
 
+/**************************************
```

Comment:
why is that comment block added here? Shouldn't it be moved a couple of lines down (after #define MY_CONTROLLER_IP_ADDRESS)